view-user/src/components/common/Header/Header.module.css

```
@@ -0,0 +1,17 @@
+.container {
+    height: 6rem;
+    width: 100vw;
+    background-color: #D9D9D9;
+    border-bottom: 0.3rem solid #333;
+}
+
+.title {
+    color: #333;
+    font-size: 4.5rem;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 0.7;
+    margin: 1rem;
+    display: inline-block;
+
+}
```

```suggestion
.title {
    color: #333;
    font-size: 3.5rem;
    font-style: normal;
    font-weight: 700;
    line-height: 1;
    margin: 1rem;
    display: inline-block;

}
```
ヘッダーにページ遷移するボタンとかログアウト入れるとおもうから少しサイズ小さくてもいいかも
変えたくなかったらそのままでいいです。